src/utils.py

```diff
@@ -200,10 +199,7 @@ def server_run_flow(flow_name, flow_args):
     """
     logger.info('Running flow {}'.format(flow_name))
     start = datetime.datetime.now()
-
-    init_celery(result_backend=False)
```

Comment:
@abs51295 as per our conversation, please restore this.

Reply:
Sure :)